nss-lean-anteaters/nutshell
# added file system and edited tag builder function for better clarity

## Description of Proposed Changes

added new file system, made edit to tag-builder function and comments for better clarity. 

## Steps to Test


```sh
git fetch --all
git checkout dpbranch

```
check to see that new directory structure and blank files representing future functions are present


## Impacted Areas in Application

List general components of the application that this PR will affect:

* anything previously built with tag builder function might need to change requirement directory
